scripts/tools/linux-install-bazel

```diff
@@ -8,31 +8,32 @@ fi
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
 . ${DIR}/all-utilities || { echo "Cannot load Bash utilities" ; exit 1 ; }
 
-BAZEL_VERSION='0.28.0'
-BAZEL_BASE_URL='https://github.com/bazelbuild/bazel/releases/download'
-BAZEL_SH="bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh"
-BAZEL_URL="${BAZEL_BASE_URL}/${BAZEL_VERSION}/${BAZEL_SH}"
-BAZEL_DIRECTORY="${TOOLS_DIR}/bazel"
+# Use Bazelisk launcher for Bazel (automatically downloads required version of Bazel).
+BAZELISK_VERSION="v1.0"
+BAZELISK_BASE_URL="https://github.com/bazelbuild/bazelisk/releases/download"
+BAZELISK_BIN="bazelisk-linux-amd64"
+BAZELISK_URL="${BAZELISK_BASE_URL}/${BAZELISK_VERSION}/${BAZELISK_BIN}"
+BAZELISK_DIRECTORY="${TOOLS_DIR}/bazelisk"
 
 function update_bazel() {
-  local bazel_current="$(bazel version | grep -oh '[0-9]*\.[0-9]*\.[0-9]*')" \
+  local bazel_current="$(bazel version | grep -oh 'Bazelisk version: v[0-9]*\.[0-9]*')" \
```

Comment:
```suggestion
  local bazel_current="$(bazelisk version | grep -oh 'Bazelisk version: v[0-9]*\.[0-9]*')" \
```

Reply:
No, it's installed as `/usr/local/bin/bazel`.

Reply:
Okay, I see.